.github/workflows/WhenPR.yml

```diff
@@ -0,0 +1,42 @@
+name: Run Test Android 
+
+on:
+  pull_request:
+    types: [opened,reopened,edited,synchronize]
+    branches: [ "main" ]
+  workflow_dispatch:
+jobs:
+  Test:
+
+    runs-on: ubuntu-latest
+
+    
+    steps:
+      - name: Fail if draft PR
```

Comment:
I don't think this should be configured here, the pipeline shouldn't even be launched if the PR is a draft. This is time and resource consuming.

Reply:
types: [opened,reopened,edited,synchronize]

This is when the test is trigged, this includes when is a draft.
The fastest way I found is check if it's a draft and mark it as failed
If I Just Check if it's not a draft it skipped it shown as everything is ok...

Reply:
Probably this can be configured in the repo settings.

Reply:
No idea where to change it in config ... but let's remove it and create a task and don't forget to optimice it

Reply:
I have changed something, now it supposed to be triggered if the code has changes. This message is a test too..